src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -710,14 +713,13 @@ def __export_features(retrieved_features, naming_convention):
 
             elif feature.state == "conditional":
                 feature_state = {feature_reserved_keywords.enabledfor: []}
-                client_filters = feature.conditions["client_filters"]
-                # client_filters is a list of dictionaries, where all dictionaries have 2 keys - Name and Parameters
-                for filter_ in client_filters:
-                    feature_filter = {}
-                    feature_filter["Name"] = filter_.name
-                    if filter_.parameters:
-                        feature_filter["Parameters"] = filter_.parameters
-                    feature_state[feature_reserved_keywords.enabledfor].append(feature_filter)
+
+                for condition_key, condition in feature.conditions.items():
+                    if condition_key == FeatureFlagConstants.CLIENT_FILTERS:
+                        feature_state[feature_reserved_keywords.enabledfor] = [filter_.to_dict() for filter_ in condition]
+                        continue
```

Comment:
We can use if/else statement instead of loop control statements like `continue`.

Reply:
This has been updated, thanks!